lib/alchemy/upgrader/tasks/available_contents_upgrader.rb

```diff
@@ -9,6 +9,10 @@ class AvailableContentsUpgrader < Thor
     no_tasks do
       def convert_available_contents
         config = read_config
+        unless config
```

Comment:
This will almost never be the case, as an empty `yml` file will be an empty array `[]`, which is not falsey, correct?